databricks/koalas/tests/test_series.py

```diff
@@ -188,6 +188,14 @@ def test_last(self):
             self.kser.last("1D")
         self.assert_eq(ks_input.last("1D"), pd_input.last("1D"))
 
+    def test_first(self):
+        index = pd.date_range("2018-04-09", periods=4, freq="2D")
+        pd_input = pd.Series([1, 2, 3, 4], index=index)
+        ks_input = ks.Series([1, 2, 3, 4], index=index)
```

Comment:
Shall we use `ks.from_pandas`?
Could you also update it in `test_last`?

Reply:
Btw, we usually use `pser` or `kser` for `Series` in tests. Could you follow it?

Reply:
Sure will make the change :) Apologies for the inconsistency!

Reply:
No worries! Thanks for the change!